src/components/ebay-menu-button/README.md

```diff
@@ -32,6 +32,7 @@ Name | Type | Stateful | Required | Description
 `checked` (radio) | Number | Yes | No | will set the corresponding index item to `checked` state and use the `aria-checked` attribute in markup
 `disabled` | Boolean | Yes | No | Will disable the entire dropdown (disables the ebay-button label) if set to true
 `variant` | String | No | No | will change the button style, "overflow" / "default"
+`collapseOnSelect` | Boolean | Yes | No | Will collapse whole menu when an item is selected in menu. Only works when `type="radio"`
```

Comment:
Nice. I like the fact it's behind a flag and backwards compatible. Given that fact, maybe we should open it up to all menuitem types, and not just radio?

Reply:
I added it in two places because I want to maintain not collapsing on radio menu if the selected item is checked.